docs/config/lua/wezterm/action_callback.md

```diff
@@ -0,0 +1,39 @@
+# `wezterm.action_callback(callback)`
+
+*Since: nightly*
```

Comment:
nit: I tend to use this phrasing so that I can more easily grep to find things when I make a release:

```suggestion
*Since: nightly builds only*
```